openssl-sys/build/run_bindgen.rs

```diff
@@ -119,14 +125,19 @@ pub fn run_boringssl(include_dirs: &[PathBuf]) {
         .enable_function_attribute_detection()
         .default_macro_constant_type(MacroTypeVariation::Signed)
         .rustified_enum("point_conversion_form_t")
-        .allowlist_file(".*/openssl/[^/]+\\.h")
+        .allowlist_file(".*[/\\\\]openssl/[^/]+\\.h")
```

Comment:
Don't the trailing separators need to be adjusted too?

Reply:
Great question. This part I just cargo culted from #2089

Reply:
No, I think the trailling separator comes directly from the include directive of the boringssl headers like `#include <openssl/cipher.h>`. The windows path will be like `\path\to\boringssl\src\include\openssl/cipher.h`.